dashboard/src/main/resources/config/liquibase/changelog/20230531-add-dashboard-table.xml

```diff
@@ -80,10 +80,20 @@
         </addColumn>
     </changeSet>
 
-    <changeSet id="5" author="Ritense">
+    <changeSet id="6" author="Ritense">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="postgresql"/>
+        </preConditions>
+        <dropNotNullConstraint tableName="dashboard_widget_configuration"
```

Comment:
Will this break if there is no constraint?

Reply:
fixed